### PR TITLE
Test Lena D

### PR DESCRIPTION
**Pull request by LenaDik**
Test https://www.lafitness.com/

### CODE REVIEW

**Review by Lizzzzi — APPROVED** [review]
agree!

**Review by valdisone — APPROVED** [review]
LGFM

File: src/test/java/GroupBreakingBadJavaTest.java [hunk 1]
```diff
@@ -67,11 +67,24 @@ public void evgeniyTMainGitCommandTest() {
         Assert.assertEquals(result.getText(), "Основные Git команды.");
     }
 
+    @Test
+    public void lenaDikTest(){
```
Comment (guram21) [inline src/test/java/GroupBreakingBadJavaTest.java:71]: All methods with `@Test` annotation shall start with the prefix `test`.
`testLenaDik()`

**Review by guram21 — CHANGES_REQUESTED** [review]

File: src/test/java/GroupBreakingBadJavaTest.java (hunk 1 above)
Comment (LenaDik) [reply src/test/java/GroupBreakingBadJavaTest.java]: fixed, please check!

**Review by guram21 — APPROVED** [review]

File: src/test/java/GroupBreakingBadJavaTest.java [hunk 2]
```diff
@@ -120,23 +120,4 @@ public void testLenaDik(){
...
-        js.executeScript("arguments[0].style.backgroundColor= '" + color + "' ", element);
-
-        try {
-            Thread.sleep(20);
-        } catch (InterruptedException e) {
-            System.out.println("Something went wrong.");
-        }
-    }
-}
+}
```
Comment (SergeiKuznetsov) [inline src/test/java/GroupBreakingBadJavaTest.java:123]: @guram21 последнюю строку зря удалили

**Pull request merged**